example/src/main/java/org/aerogear/mobile/example/ui/SecurityServiceFragment.java

```diff
@@ -81,15 +86,25 @@ public void onActivityCreated(@Nullable final Bundle savedInstanceState) {
      * Executes all test and calls setTrustScore() to calculate an average score
      */
     public void runTests() {
+        SyncSecurityCheckExecutor executor = (new SyncSecurityCheckExecutor.Builder(this.getContext()))
```

Comment:
Change this to 
`SecurityCheckExecutor.newSyncExecutor(this.getContext())`

Reply:
@ziccardi Is there a way to provide a reference to the metrics service this way?

See https://github.com/aerogear/aerogear-android-sdk/pull/116#discussion_r171285424.

Reply:
@paolobueno yes See here https://github.com/aerogear/aerogear-android-sdk/blob/master/auth/src/test/java/org/aerogear/mobile/security/SecurityCheckExecutorTest.java#L53

Reply:
Got it, changing. Any reason for the extra method hop though?

Reply:
Not really. Feel free to fix that.